src/molecules/header-card/header-card.tsx

```diff
@@ -37,8 +37,20 @@ const HeaderCardContainer = styled.div`
 export const DefaultHeaderCard = ({ story, config, storyType }: HeaderCardProps) => {
   const { publisherConfig } = config;
 
+  const theme = useTheme();
+
   const { enableLastPublished, enableFirstPublished } = getDateSettings(config, storyType);
 
+  function authorCardRender(data) {
+    let authorUi;
+    const authorCard = get(config, ["opts", "featureConfig", "renderAuthorCard"], null);
+
+    if (authorCard) {
+      authorUi = authorCard({ ...data, theme });
```

Comment:
I'd suggest not spreading `data` and passing it as separate keys so that we're sure what all params the render prop offers to the end user